src/artofillusion/tapDesigner/TaPDMetalTheme.java

```diff
@@ -1,79 +0,0 @@
-/*
```

Comment:
Probably shouldn't delete this file quite yet. It is referenced in `tapDesigner/TaPDPreferences`. Does it actually cause compilation issues?

Reply:
Probably should delete TaPDPreferences class as well as this one. The TaPDPreferences  is not used in code...

Reply:
Probably. It looks like it only ever played with the theme a bit...